### PR TITLE
Controller refactor#145

### LINKED ISSUE #145: Move ActiveRecord call out of merchant/orders_controller show method

### PR DESCRIPTION
**Pull request by timnallen**
Refactored the controller by adding an order model instance method ordered items from merchant complete with test!

closes #145

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -7,6 +7,12 @@ class Order < ApplicationRecord
 
   enum status: ['pending', 'processing', 'completed', 'cancelled']
 
+  def ordered_items_from_merchant(merchant)
```
Comment (teresa-m-knowles) [inline app/models/order.rb:10]: good refactor!

**Review by teresa-m-knowles — APPROVED** [review]
Like the refactor!

**Review by allieIsOkay — APPROVED** [review]
Looks good, thanks Tim.

**Review by csvlewis — APPROVED** [review]
Good refactor!

**Pull request merged**